### PR TITLE
Histogram Example

### PR DESCRIPTION
**Pull request by JhaShweta1**
# Description

WIP- PR for Histogram example.

Fixes # (issue) 

## Type of change

Please delete options that are not relevant. Add a 'X' to the one that is applicable. 

- [ ] Bug fix (non-breaking change which fixes an issue)
- [X ] New feature (non-breaking change which adds functionality)
- [ ] Sample Migration (Moving sample from old repository after completing checklist established)

# How Has This Been Tested?

Please describe the tests that you ran to verify your changes. Provide instructions so we can reproduce. Please also list any relevant details for your test configuration

- [X] Command Line
- [ ] oneapi-cli
- [ ] Visual Studio
- [ ] Eclipse IDE
- [ ] VSCode

# Checklist for Moving samples:
Links and Details can be found in the samples WG Teams Files. 

- [ ] Review sample design with domain reviewers https://github.com/oneapi-src/oneAPI-samples/wiki/Reviewers-and-Domain-Experts 
- [ ] Implement coding guidelines and ensure code quality.
- [X] Adhere to sample.json specification. https://github.com/oneapi-src/oneAPI-samples/wiki/sample-json-specification
- [X] Run jsonlint on sample.json to verify json syntax. www.jsonlint.com
- [X] Adhere to readme template 
- [ ] Ensure/create CI test configurations for sample (ciTests field) https://github.com/oneapi-src/oneAPI-samples/wiki/sample-json-ci-test-object
- [ ] Enforce format via clang-format config file
- [X] Review DPC++ code with Paul Peterseon. (GitHub User: pmpeter1)
- [ ] Review readme with Tom Lenth or Joe Oster. (GitHub User: JoeOster)
- [ ] Tested using Dev Cloud when applicable
- [ ] Implement fixes for ONSAM Jiras
- [ ] If you have new dependencies/binaries, inform Samples Project Manager Swapna R Dontharaju (@srdontha) or @JoeOster

### CODE REVIEW

**Comment by JoeOster** [comment]
> In regards to your readme.md, please include a statement about where the dpc_common file is located, such as the statement within the mandelbrot readme.md file,
> https://github.com/Alberto-Villarreal/oneAPI-samples/tree/master/DirectProgramming/DPC%2B%2B/CombinationalLogic/mandelbrot

@JhaShweta1 -Please take a look at this

**Comment by JhaShweta1** [comment]
@JoeOster I have updated the Readme.md.

File: DirectProgramming/DPC++/ParallelPatterns/histogram/src/main.cpp [hunk 1]
```diff
@@ -0,0 +1,111 @@
+//==============================================================
+// Copyright © 2020 Intel Corporation
+//
+// SPDX-License-Identifier: MIT
+// =============================================================
+
+#include <CL/sycl.hpp>
+#include "dpc_common.hpp"
```
Comment (JoeOster) [inline DirectProgramming/DPC++/ParallelPatterns/histogram/src/main.cpp:8]: dpc_common.hpp should always be last in the list
Also need to add a comment in regards to location, should look like


// dpc_common.hpp can be found in the dev-utilities include folder.
// e.g., $ONEAPI_ROOT/dev-utilities/<version>/include/dpc_common.hpp
#include "dpc_common.hpp"

**Comment by JoeOster** [comment]
> @JoeOster I have updated the Readme.md.

Thank you

**Comment by JoeOster** [comment]
@JhaShweta1 JhaShweta1 - can you please take a look at the compile errors

**Comment by JoeOster** [comment]
> @JhaShweta1 JhaShweta1 - can you please take a look at the compile errors

Compile errors are due to changes between Beta08 and Beta09 compilers

**Comment by JhaShweta1** [comment]
@JoeOster I have got the access to the machine where CI is run. I still see some discrepancies in the output of different machines where I run the sample. I will look more into the error this week.

**Pull request merged**